docs/src/content/docs/challenges/angular/48-dialog-alert-form.md

```diff
@@ -0,0 +1,44 @@
+---
+title: 🟠 Save your form&#39;s bacon: with the navigation alert
+description: Challenge 48 is about Bob 🧙‍♂️ the product owner, he wants to develop a new feature in response to customer complaints about losing form input information.
+author: Timothy Alcaide
```

Comment:
you need to create a file inside the author folder and copy the name of this file here.